src/node/util.ts

```diff
@@ -241,8 +249,71 @@ export function getSystemErrorMessage(): void {
   throw new Error('node:util getSystemErrorMessage is not implemented');
 }
 
-export function styleText(): void {
-  throw new Error('node:util styleText is not implemented');
+function escapeStyleCode(code: number | undefined): string {
+  if (code === undefined) return '';
+  return `\u001b[${code}m`;
+}
+
+// TODO(gbedford): We do not yet implement process.stdout, so to ensure the correct
```

Comment:
We don't use username todos, but always prefer TODO(soon) for things that are prioritized. All other TODOs are from node.js codebase which we deliberately leave as is to keep code difference in minimum

```suggestion
// TODO(soon): We do not yet implement process.stdout, so to ensure the correct
```